yarn.lock

```diff
@@ -7878,7 +7878,7 @@ available-typed-arrays@^1.0.5:
 
 "aws-iot-twinmaker-grafana-utils@file:libs/aws-iot-twinmaker-grafana-utils-2.0.0.tgz":
   version "2.0.0"
-  resolved "file:libs/aws-iot-twinmaker-grafana-utils-2.0.0.tgz#cb39fe168a7877e9efb3188ffeadcb5d6bad4819"
+  resolved "file:libs/aws-iot-twinmaker-grafana-utils-2.0.0.tgz#74ef4926dd99a7e7e96c65a868e3a57671d876e4"
```

Comment:
Why does the version number stay the same when the hash changes?

Reply:
We usually don't change the file name unless its a major version change this was a minor version change from 2.0.22 to 2.0.29.